### PR TITLE
Add VIA support for YD60MQ

### PR DESCRIPTION
**Pull request by rbange**
## Description

This PR adds VIA support for the `YD60MQ` board, as last PR https://github.com/qmk/qmk_firmware/pull/10273 has been dropped.

VIA-PR: https://github.com/the-via/keyboards/pull/309

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/yd60mq/config.h [hunk 1]
```diff
@@ -3,8 +3,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x594D // "YM"
+#define PRODUCT_ID      0x604D // 60M
```
Comment (fauxpark) [inline keyboards/yd60mq/config.h:7]:
```suggestion
#define PRODUCT_ID      0x604D // 60 "M"
```

**Review by fauxpark — APPROVED** [review]

File: keyboards/yd60mq/config.h (hunk 1 above)
Comment (rbange) [reply keyboards/yd60mq/config.h:7]: Sure 👍

File: keyboards/yd60mq/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
```
Comment (noroadsleft) [inline keyboards/yd60mq/keymaps/via/keymap.c:1]: Could you add a license header here?

Here's a sample one:

https://github.com/qmk/qmk_firmware/blob/0b2eae768637bc9230fec5d5167f617b21f4e179/quantum/template/base/keymaps/default/keymap.c#L1-L15

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/yd60mq/keymaps/via/keymap.c (hunk 2 above)
Comment (rbange) [reply keyboards/yd60mq/keymaps/via/keymap.c:17]: Sure, just added a license in https://github.com/qmk/qmk_firmware/pull/10321/commits/7bd646ef1ff0ae18d508dc9c470d74c799d0f1f7

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**